src/main/java/oc/RuestkammerVater.java

```diff
@@ -25,6 +26,10 @@
 	private JButton ok = new JButton("OK");
 	private JLabel error = new JLabel();
 	private JFrame frame = new JFrame();
+    protected String defaultNK = "";
+    protected String defaultFK = "";
```

Comment:
Rename Properties to `defaultCloceCombat` and `defaultRanged` to increase readability and try to move the codebase over to english where possible.